src/handle_copymove.rs

```diff
@@ -84,7 +84,7 @@ impl crate::DavInner {
             // If we encounter errors, just print them, and keep going.
             // Last seen error is returned from function.
             let mut retval = Ok::<_, DavError>(());
-            while let Some(dirent) = entries.next().await {
+            while let Some(dirent) = entries.next().await.transpose()? {
```

Comment:
Instead of tranpose `Result` into `Option`, would it make more sense to handle the `Err` case here and in other similar code paths?

Reply:
Fixed, PTAL.